pyproject.toml

```diff
@@ -0,0 +1,91 @@
+[build-system]
+build-backend = "mesonpy"
+requires = [
+    "meson-python",
+    "numpy",
+    "wheel",
+    "pybind11",
+]
+
+[project]
+name = "meshpy"
```

Comment:
Hm, I think the project name was `MeshPy` before. Should rename it back?

Reply:
These things are not case-sensitive, so :shrug:. :)